pom.xml

```diff
@@ -52,6 +55,40 @@
     <plugin.maven.enforcer.version>3.0.0</plugin.maven.enforcer.version>
   </properties>
 
+  <profiles>
+    <profile>
+      <id>release</id>
+      <build>
+        <pluginManagement>
+          <plugins>
```

Comment:
Looking at the requirements to release to maven central: https://central.sonatype.org/publish/requirements/, it points to a need for javadocs and source jars to be included. Consider adding the `maven-source-plugin` and `maven-javadoc-plugin` to this profile.

Reply:
There may be other requirements from central at that link that'll be needed. It looks like a `<description>` (usually next to `<name>` and `<url>`) is required as well.

Reply:
the javadoc and source plugins have been added, as well as a description